build-tools/automation/yaml-templates/setup-test-environment.yaml

```diff
@@ -10,13 +10,19 @@ parameters:
   restoreNUnitConsole: true
   updateMono: true
   androidSdkPlatforms: $(DefaultTestSdkPlatforms)
+  repositoryAlias: 'self'
+  commit: 'HEAD'
```

Comment:
Can we use an empty string here by default instead, like we're doing in the [build templates](https://github.com/xamarin/xamarin-android/blob/main/build-tools/automation/yaml-templates/build-macos.yaml#L5)? Same thing applies to the other templates